conans/test/integration/command/runner_test.py

```diff
@@ -0,0 +1,222 @@
+import textwrap
+import os
+import pytest
+import docker
+from conans.test.utils.tools import TestClient
+from conans.test.assets.cmake import gen_cmakelists
+from conans.test.assets.sources import gen_function_h, gen_function_cpp
+
+
+
+def docker_skip(test_image=None):
+    try:
+        docker_client = docker.from_env()
+        if test_image:
+            docker_client.images.get(test_image)
+    except docker.errors.DockerException:
+        return True
+    except docker.errors.ImageNotFound:
+        return True
+    except docker.errors.APIError:
+        return True
+    return False
+
+
+def conan_base_path():
+    import conans
+    return os.path.dirname(os.path.dirname(conans.__file__))
```

Comment:
Same comment as the other, you may use `client.current_folder` if you save the dockerfile from a string.

Reply:
This is to get the source path of all the current conan code to copy it into the container and have this feature implemented without push and clone the code inside the docker build steps.